lms/static/js/spec/learner_dashboard/progress_circle_view_spec.js

```diff
@@ -0,0 +1,126 @@
+define([
+    'backbone',
+    'jquery',
+    'js/learner_dashboard/views/progress_circle_view'
+], function(Backbone, $, ProgressCircleView) {
+    'use strict';
+
+    describe('Program Progress View', function() {
```

Comment:
Nit: this is no longer a program test, so it should be renamed "Progress Circle View".